copier.yml

```diff
@@ -147,11 +136,11 @@ services_repo:
 
 services_release:
   type: str
-  help: The initial release of the services repository to track e.g. "2024.1.1"
+  help: The initial release of the services repository to track e.g. "2024.12.1" or "main"
```

Comment:
Im second guessing the use of "initial". Should updates to the release not be done by a copier update as best practice?

Reply:
I dont think so - this is just what you will start with. I think that is what Initial means ?

Reply:
Yes that is currently what is implied - but the question is should it? Should we not encourage updating the release rather through a copier update?

Reply:
I feel that dirties the idea of what the template is for. But that's because I've not considered using this way before. But I think I prefer the idea that ec updates what we track, the template updates what the generic parts of the repo look like.

Reply:
Thats fine. Its not used in more places so copier probably wont break. I imagine ec only updating on a service level though so its two different places